src/elements/common/sub-header/SubHeader.js

```diff
@@ -50,7 +54,9 @@ const SubHeader = ({
             canCreateNewFolder={canCreateNewFolder}
             canUpload={canUpload}
             currentCollection={currentCollection}
+            isGridView={isGridView}
             onCreate={onCreate}
+            onGridViewSwitch={onGridViewSwitch}
```

Comment:
We typically use `toggle` for switching between states, so this could be `onGridViewToggle`. However, it could also be modified to support multiple other "views", in case we add more in the future. In that case, something like `onViewChange` might make sense.